python/distant_vfx/parsers.py

```diff
@@ -68,10 +68,13 @@ def _drop_data_header_row(self):
         self.column_data = self.column_data[self.column_data.Name != 'Data']
 
     def _rename_columns(self):
+        def column_replacer(column_name):
```

Comment:
This might want your eyes as well for it's replacer code.

I think your lambda replaced every character in `forbidden_chars` with an underscore. 
The implementation that you did was a bit unclear, so I may be wrong with what you were doing.

I think my version is more clear and it supports python2

Reply:
Yep, that's what I was doing here. One thing - `repplace` should be `replace` right?

Reply:
You betcha :P